### PR TITLE
Add `isNewAction` and `isRepositoryActions`

### LINKED ISSUE #40: Add `isNewActionRun`
https://github.com/sindresorhus/refined-github/actions/new

`isNewActionRun`?
`isNewAction`?

### PR DESCRIPTION
**Pull request by yakov116**
Closes #40 

Blocked by #42

### CODE REVIEW

**Comment by yakov116** [comment]
RRR I wish I would be able to test local /endrant

**Comment by yakov116** [comment]
@fregante help! What is wrong?

**Comment by fregante** [comment]
The `pull_request` jobs are run on the merge commit, which you cannot see. You need to updated the branch ⬆️

File: index.ts [hunk 1]
```diff
@@ -453,6 +453,17 @@ collect.set('isActionJobRun', [
 	'https://github.com/sindresorhus/refined-github/runs/639481849',
 ]);
 
+export const isNewAction = (url: URL | Location = location): boolean => String(getRepoPath(url)) === 'actions/new';
+collect.set('isNewAction', [
+	'https://github.com/sindresorhus/refined-github/actions/new',
+]);
+
+export const isRepositoryActions = (url: URL | Location = location): boolean => isRepo() && String(getRepoPath(url)).startsWith('actions');
```
Comment (fregante) [inline index.ts:461]: `getRepoPath` already includes `isRepo`

**Comment by yakov116** [comment]
Thanks!

Any gotcha's need to be added?

**Comment by fregante** [comment]
Nah, it's super simple.

**Comment by fregante** [comment]
> @fregante help! What is wrong?

I just noticed, you were calling `isRepo()` instead of `isRepo(url)`

**Comment by yakov116** [comment]
> > @fregante help! What is wrong?
> 
> I just noticed, you were calling `isRepo()` instead of `isRepo(url)`

Ahh!

**Comment by fregante** [comment]
> RRR I wish I would be able to test local /endrant

Why can't you? Did you run out of gigabytes? 😄

**Review by fregante — APPROVED** [review]

**Comment by yakov116** [comment]
> > RRR I wish I would be able to test local /endrant
> 
> Why can't you? Did you run out of gigabytes? 😄

Something with a ts-node issue

**Comment by fregante** [comment]
Anything else needed before I tag v4?

**Comment by yakov116** [comment]
Not that I saw.

**Comment by yakov116** [comment]
WHY CANT I FOLLOW INSTRUCTIONS??? 

>Totally. This should be named `isMarketplaceAction` (not `s`, it's just one)

_Originally posted by @fregante in https://github.com/fregante/github-url-detection/issues/41#issuecomment-701575539_

**Comment by fregante** [comment]
You followed that one. And this one is correct “Actions” as in “the Actions tab”

It’s not a single action in `isRepositoryActions`

**Comment by fregante** [comment]
I think this function should be more specific. It makes no sense to bundle “actions/runs” and “actions/“ together, they’re 2 different pages. 

Instead of startsWith, it should just use `=== 'actions'`

**Comment by yakov116** [comment]
I should have double checked first. I incorrectly assumed that going to a workflow was going to be actions/workflowname.
I just checked now it's a searchparam so you are correct.
https://github.com/sindresorhus/refined-github/actions?query=workflow%3ADeployment

**Comment by fregante** [comment]
I mean even the workflow has a different page layout, so it’s not really useful for us to lump it with the Actions list

**Comment by yakov116** [comment]
Agree. Would you like me to change this one and make one for a workflow?

**Comment by fregante** [comment]
No worries, I didn't check either. 

What page is "a workflow"? I don't think we're using that anywhere.

Yes you can just change this and update the tests, no worries even if it's a "breaking" change

**Comment by fregante** [comment]
> Curious why would this be different than `isPr`?

How is it related?

**Comment by yakov116** [comment]
We don't use it anywhere (yet).

**Comment by yakov116** [comment]
> > Curious why would this be different than `isPr`?
> 
> How is it related?

I would think the `isRepositoryActions` would give me all pages for actions. Just like we have `isPr` and then `isConversation` for that specific page.

**Comment by fregante** [comment]
`/pulls/1/*` pages share the PR header

`/action/*` pages don't, they're completely different `isRepo` pages (i.e. they only share the repo header)

**Pull request merged**